### DISCUSSION TITLE
Start xcom 2 with aml, it will change from Chinese to English and crash

### DISCUSSION THREAD

**Discussion by anbai1** [X2CommunityCore/xcom2-launcher#330] [Help] [answered]
Start playing logo, then load it into the "loading..." screen JPG, and start to crash.

MODs within 10 will be good, and 60 mods will crash.
However, it is no problem to use the 2K launcher, even if 60 mods are loaded.
Now, if there is only one mod or no mod, aml cannot be used.
I prefer aml, so I need help.
Before that, my aml was broken.
I reinstall win10, xocm2, and all mods.
But now, the same error as before the reinstallation.
Sorry, this is machine translation.

file and screenshots :
https://drive.google.com/drive/folders/1zw6wNFY25oKE82D3w7gz0ffaYGz-81sF?usp=sharing

thank you.

**Comment by anbai1** [comment, marked as answer]
I may have found a solution.
AML runs as an administrator of Windows.
I'm running the game in Chinese.
Currently running well.

**Reply by RevZero** [reply]
I am not sure I really understood your problem, so I am not sure if/why running as admin would solve the problem.
However, glad to hear it worked for you.
